src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -296,20 +321,28 @@ class PrisonerIepLevelReviewService(
     incentiveLevels: Map<String, IepLevel>,
     withDetails: Boolean = true
   ): IepSummary {
-    val iepLevels = levels.map { it.translate(incentiveLevels) }
+    val iepDetails = levels.map { it.translate(incentiveLevels) }.toList()
 
-    val currentIep = iepLevels.firstOrNull() ?: throw IncentiveReviewNotFoundException("Not Found incentive reviews")
+    val currentIep = iepDetails.firstOrNull() ?: throw IncentiveReviewNotFoundException("Not Found incentive reviews")
```

Comment:
Unfortunately it's not possible because. `iepDetails` is passed in and this would only happen when `iepDetails` is empty and therefore the method has no way to know which `bookingId` this is for. If it makes sense :)